### DISCUSSION TITLE
Ватрас счётчик убийств

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#485] [Q&A]
Мы этот вопрос поднимаем уже каждый год :-) по моему ты мне говорила что Ватрас считает убийства Героя в 5 главе. Давай ещё раз посмотрим как должен считать Ватрас и как он фактически считает.

Я как понимаю по ЛОРу он должен считать убийства в 5 главе. ну или в 4-5 главе.

**Comment by dosinabox** [comment]
Переменная MadKillerCount:
- используется в книге статистики для показа количества убийств невинных людей
- увеличивается на 1 при убийстве каждого невинного человека (с первой главы)
- проверяется Ватрасом (с пятой главы)
- уменьшается на 1 за каждые 10 благословений Ватраса (с первой главы)
- перестает уменьшаться за благословения, если Ватрас сделал предупреждение за 7 убитых (с пятой главы)

**Comment by Kor-Angar** [comment]
> уменьшается на 1 за каждые 10 благословений Ватраса (с первой главы)

которые бесплатные или за рассказ ГГ ?

так же возможно имеет смысл обнулять при вступление в КВ.

**Comment by Kor-Angar** [comment]
> MadKillerCount

бандиты, пираты, наёмники и ОнД считаются как?

**Comment by dosinabox** [comment]
> которые бесплатные или за рассказ ГГ ?

За рассказ, бесплатные, и за каждые 100 золотых.

> так же возможно имеет смысл обнулять при вступление в КВ.

Почему?

> бандиты, пираты, наёмники и ОнД считаются как?

Невинные люди - те, которые падают в нокаут и которых требуется добивать.

**Comment by Kor-Angar** [comment]
> Невинные люди - те, которые падают в нокаут и которых требуется добивать.

это тогда все люди :-)

***

> Почему?

так по ЛОРу по моему идёт. КВ это же аналог монастыря магов огня.

***

по моему Ватрас еще благословение даёт за спасение людей.

**Comment by dosinabox** [comment]
> это тогда все люди :-)

Бандиты и ищущие сразу погибают, без нокаута. Они не считаются невинными.
Ещё погибают сразу и невинными не считаются обладатели аивара AIV_DropDeadAndKill. Это Сильвио в Минентале, охранник портового склада, Рик, Румболд, Рамирез, Джеспер, Кассия, Аттила, Ханна после уничтожения гильдии и Игарац, Агон и Ульф на время испытания.

**Comment by Kor-Angar** [comment]
я это понимаю. но я не понимаю почему нападает Сильвио и Булко. При этом сильвио "виновный", а Булко "невиновный" :-)

**Comment by dosinabox** [comment]
С Буллко потом диалоги есть)

**Comment by Kor-Angar** [comment]
я же про ЛОР и про виновных и невиновных. в целом на мой взгляд "невинные" это крестьяне и горожане :-)
